src/slider/slick/track.jsx

```diff
@@ -161,17 +199,20 @@ const renderSlides = (specProps) => {
 export default class Track extends Component {
     static propTypes = {
         prefix: PropTypes.string,
-        trackStyle: PropTypes.object
+        trackStyle: PropTypes.object,
     };
 
     static defaultProps = {
-        prefix: 'next-'
+        prefix: 'next-',
     };
 
-    render () {
+    render() {
         const slides = renderSlides(this.props);
         return (
-            <div className={`${this.props.prefix}slick-track`} style={this.props.trackStyle}>
+            <div
```

Comment:
if the slides have `role="listitem"` then this should be `ol`, `ul`, or `role="list"`